### DISCUSSION TITLE
Need help setting up a electricity "mini graph's" for days, months, years, and comparing X years

### DISCUSSION THREAD

**Discussion by DuckersMcQuack** [kalkih/mini-graph-card#1079] [General]
As i'm not good with code as it can quite easily overwhelm me, can someone help me setup cards to look like picture below?

As i'd want 4 different cards, as history tab for power in HA is quite limited for sensors it can access for power draw, as the only one it can read is quite wrong for hourly readings, but cards can read from any sensors, so i want to setup 4 or 5 different cards for electricity readings

I'd want them to read say every hour of 1 day with power draw scale on left vertical, power cost entity on the right scale, and bars in the middle for current consumption with hours noted below each graph.
Doesn't need every hour mentioned in the code, just an example "hour 00:00" and "hour:01:00, and i'll write the rest from the template.
Same with days of the week, and weekly, monthly, and yearly basis. and if possible, a double graph one that shows 2 graph bars next to eachother per month to compare last year and current year with top middle stating orage being current year, grey was last year.

this is the kind of graph i want for each of the cards. Orange shows price, blue is power used.

![image](https://github.com/kalkih/mini-graph-card/assets/9039370/36dffd2b-94a4-414b-9ac4-724378f5f182)

**Comment by ildar170975** [comment]
This is not a bug or a FR, this is a question.
Suggest to ask here:
https://community.home-assistant.io/t/lovelace-mini-graph-card/71385/
At least you will attract more people.

**Reply by DuckersMcQuack** [reply]
Thanks! I shall :P Thanks for pointing fingers where to ask instead!
